code/modules/clothing/glasses/glasses.dm

```diff
@@ -295,15 +296,41 @@ BLIND     // can't see anything
 	flip_down = "up to protect your eyes."
 	flip_up = "and let it hang around your neck."
 
+//essentially just sunglasses
+/obj/item/clothing/glasses/safety/goggles/tactical
+	name = "tactical goggles"
+	desc = "A stylish pair of tactical goggles that protect the eyes from aerosolized chemicals, debris and bright flashes."
```

Comment:
```suggestion
	desc = "A stylish pair of tactical goggles that protect the eyes from aerosolized chemicals, debris, and bright flashes."
```

Reply:
I don't think serial commas are common here, so I won't be doing that unless we're required to follow American English.